mne/report.py

```diff
@@ -1414,6 +1456,33 @@ def _render_cov(self, cov_fname, info_fname):
                                          show=show)
         return html
 
+    def _render_whitened_evoked(self, evoked_fname, noise_cov):
+        """Show whitened evoked.
+        """
+        global_id = self._get_id()
+
+        evokeds = read_evokeds(evoked_fname, verbose=False)
+
+        html = []
+        for ev in evokeds:
+            noise_cov = regularize(noise_cov, ev.info)
+            ev_white = whiten_evoked(ev, noise_cov, diag=True)
+            global_id = self._get_id()
+
```

Comment:
Sure, we could have a parameter `True/False/'auto'`, if `auto` use the highpass information.

-1 on changing the reading default, that will make it so that if I do `write_cov(read_cov())` I get something added, which isn't great. I think we should just change our logic functions. It's more work but a little bit cleaner from an I/O perspective.